test/uk/gov/hmrc/trusts/services/DesServiceSpec.scala

```diff
@@ -279,5 +283,205 @@ class DesServiceSpec extends BaseSpec {
     }
   } //getSubscriptionId
 
+  ".getTrustInfo" should {
```

Comment:
Do we have tests where the response is `Processed` with the `Trust/Estate` and all entities returned in the response?